primitives/account/src/logs.rs

```diff
@@ -0,0 +1,364 @@
+use crate::Receipt;
+use beserial::Serialize as BeSerialize;
+use nimiq_hash::Blake2bHash;
+use nimiq_keys::Address;
+use nimiq_primitives::coin::Coin;
+use nimiq_transaction::account::htlc_contract::{AnyHash, HashAlgorithm};
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
+// Renaming affects only the struct names and thus their tag, the "type" field.
```

Comment:
Mmm, it should affect all of the members according to the documentation: _Rename all the fields (if this is a struct) or variants (if this is an enum) according to the given case convention._

Reply:
It only renames the names of the variants, not their fields too.

Reply:
Maybe it should be reworded as:
_Renaming affects only the enum variant names and its tag (the "type" field)_
?